public/index.html

```diff
@@ -5,10 +5,26 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <meta name="title" content="AsyncAPI Studio">
     <meta
       name="description"
       content="Studio for AsyncAPI specification, where you can validate, view preview documentation, and generate templates from AsyncAPI document."
     />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="/">
+    <meta property="og:title" content="AsyncAPI Studio">
+    <meta property="og:description" content="Studio for AsyncAPI specification, where you can validate, view preview documentation, and generate templates from AsyncAPI document.">
+    <meta property="og:image" content="/img/meta-studio-og-image.jpg">
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summare_large_image">
+    <meta property="twitter:url" content="/">
```

Comment:
I think that url should point to the twitter account like in conference page: 
```suggestion
    <meta property="twitter:url" content="https://twitter.com/asyncapispec">
```

and we should also add domain property:

```
    <meta property="twitter:domain" content="https://studio.asyncapi.com">
```

Reply:
So I actually think the `twitter:url` property is wrong. I think it is actually supposed to be `twitter:site` as shown here: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup

And I just noticed that I have a typo on line 22 - `summare_large_image` is supposed to be `summary_large_image` 😅 

Also, I don't think we need `twitter:domain` as I don't see it in the twitter docs. Where did you find that property?

Reply:
Ok, so it should be site :) I found domain here https://www.opengraph.xyz/url/https%3A%2F%2Fdeploy-preview-170--modest-rosalind-098b67.netlify.app%2F/ at the bottom of page:

```
<!-- Twitter Meta Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="deploy-preview-170--modest-rosalind-098b67.netlify.app">
...
```

Reply:
hmmm okay. I will make those changes!